superset-frontend/src/profile/components/UserInfo.tsx

```diff
@@ -19,17 +19,36 @@
 import React from 'react';
 import Gravatar from 'react-gravatar';
 import moment from 'moment';
-import { Panel } from 'react-bootstrap';
-import { t } from '@superset-ui/core';
+import Collapse from 'src/common/components/Collapse';
+import { t, styled } from '@superset-ui/core';
 import { UserWithPermissionsAndRoles } from '../../types/bootstrapTypes';
 
 interface UserInfoProps {
   user: UserWithPermissionsAndRoles;
 }
 
+const StyledContainer = styled.div`
+  .ant-collapse-header {
```

Comment:
Are all these styles unique to the implementation of this component for the UserInfo component, or are these styles likely to be reused? If they're likely to be reused, they should be added to the AntD component itself. If they're a _variant_ of the component, the AntD component being exported could use a `className` or `prop` to enable the styles. Then this UserInfo component can just use the AntD component out of the box, with some prop(s) or class(es) applied, if the use case / styles are repeatable.

Note that if you DO go that route, we should add a storybook entry to reflect the class(es)/prop(s)